docs/for-devs/console/services.md

```diff
@@ -24,10 +24,17 @@ All known Services of the team are listed here. Services can be sorted based on:
 
 ## Create a Service
 
-1. Enter a name for the service. The name will be used to generate hostname if `Use suggested domain` is chosen (see below). When configuring ingress for an existing (pre-deployed) service, make sure the name provided here matches the name of the CusterIP service
-2. Enter the port number of the service
+First select the (ClusterIP) service to use.
+
+## Configuring traffic control
+
+The traffic control option allows to split traffic between multiple deployed versions (blue-green, canary). Note that the traffic control can only be used when: 
+- There are 2 versions deployed that share a single service
+- The service is configured for external exposure
+
+- Select `Enabled` to configure traffic control
+- Fill in the weight for the 2 versions. A 50/50 weight can be used for blue/green. A 90/10 weight can be used for canary.
```

Comment:
Blue-green is about swapping between applications 0, 100% or if something is wrong 100, 0%
At least this is how I understand this term

Reply:
Correct. So start with 50/50 and then change to 0/100.